addons/website_slides_survey/tests/test_course_certification_remove_membership.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo.addons.survey.tests.common import TestSurveyCommon
+
+
+class TestCourseCertificationRemoveMembership(TestSurveyCommon):
+
+    def setUp(self):
+        super(TestCourseCertificationRemoveMembership, self).setUp()
```

Comment:
Hello @std-odoo 

But, here we need users and some data created from setup, and since we are using setupclass, we won't have that data.

Thanks

Reply:
it doesn't work only because of `_add_question` right ?

I would move everything that can be moved in setUpClass and keep only the `_add_question` in setUp

Reply:
Actually, we can't move anything.
since users are created from setup and we are using that user while creating Survey.
Or, we can use another user.

Reply:
the user must be created in the setUpClass imo

Reply:
Here (https://github.com/odoo/odoo/blob/master/addons/survey/tests/common.py#L267) Users are created from setup.
that's why.
either we can move all this user creation to setupclass .

Reply:
you can just rename this method setUpClass, it was a mistake I think

Reply:
Okay done